modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectFieldFormBase.tsx

```diff
@@ -107,9 +108,32 @@ export default function ObjectFieldFormBase({
 		return businessTypeMap;
 	}, [objectFieldTypes]);
 
+	const [picklistDefaultValue, setPicklistDefaultValue] = useState<
+		ObjectState
+	>();
+	const [defaultValueQuery, setDefaultValueQuery] = useState<string>('');
```

Comment:
Hey @M-IgorFranca, Could you rename this state name to `picklistDefaultValueQuery` and `setPicklistDefaultValueQuery` too?

Reply:
Its make sense, sorry!